specifyweb/frontend/js_src/lib/components/FormEditor/spec.ts

```diff
@@ -272,7 +272,10 @@ const viewDefSpec = f.store(() =>
      * formDefinitionSpec() later on
      */
     raw: syncer<SimpleXmlNode, XmlNode>(
-      (node) => defined(getOriginalSyncerInput(node), ''),
+      (node) => ({
+        ...defined(getOriginalSyncerInput(node), ''),
```

Comment:
If `getOriginalSyncerInput(node)` is undefined, would this not throw an Error with no message? 

Consider the definition of `defined`: 
https://github.com/specify/specify7/blob/48c1f00d576a03e25927847723c17be340944f80/specifyweb/frontend/js_src/lib/utils/types.ts#L37-L41

`'' ?? 'other string'` will return `''`, and `''` is probably not the most immediately helpful error message.  

Of course, ignore this if this is intentional.